### PR TITLE
CQ-4351565: added test to verify start workflow

### PR DESCRIPTION
**Pull request by egmox**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+describe('Verify Start workflow page operation on af editor', function () {
+
+    const url = '/aem/forms.html/content/dam/formsanddocuments';
+    const invokeAemWorkflowButton = '[data-item-id="fd/dashboard/components/actions/aemworkflowsubmit"]';
+    const productNavigation = 'coral-panel-content[role="presentation"] button[class="_coral-Button _coral-Button--primary _coral-Button--quiet"]';
```
Comment (nit23uec) [inline ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js:21]: pls avoid using any coral class that starts with underscore as it is internal and may change

File: ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    it('verify Start workflow page operation on af editor', function () {
+        cy.openPage(url);
+        cy.get(productNavigation).should('exist').eq(0).click();
+        cy.get('#aem-forms-create').click();
+        cy.get('[title="Adaptive Form"]').click();
+        Cypress.on('uncaught:exception', (err, runnable) => {
+            return (!err.message.includes('Handlebars is not defined'))
```
Comment (nit23uec) [inline ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js:29]: why this?

File: ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        })
+        cy.get('[role="tab"]').should('exist').eq(3).click();
+        cy.get(invokeAemWorkflowButton).click();
```
Comment (nit23uec) [inline ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js:32]: how are we verifying it workflow is actually invoked or has done its intended work?

**Review by nit23uec — CHANGES_REQUESTED** [review]
this is a editor test case. it should be internal. not in core components repo.

File: ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js (hunk 2 above)
Comment (egmox) [reply ui.tests/test-module/specs/automation/verifyStartWorkflowPageOperation.spec.js:29]: We are getting "Handlebars is not defined error" on create form page in latest cq jar which fails the test

**Pull request closed without merging**